plugin/src/main/scala-2/Scala2CompilerApi.scala

```diff
@@ -49,11 +49,13 @@ object Scala2CompilerApi {
       def addMethod(clazz: Clazz, method: Method): Clazz =
         clazz.copy(impl = clazz.impl.copy(body = clazz.impl.body :+ method))
 
-      def methodNames(clazz: Clazz): List[String] = clazz.impl.body.collect { case d: DefDef =>
-        d.name.toString
+      def methodNames(clazz: Clazz): List[String] = clazz.impl.body.collect {
+        case d: DefDef => d.name.toString
+        case d: ValDef => d.name.toString
```

Comment:
This was added to support `override val toString` in both Scala versions. A better name for `methodNames` would be welcome, although `memberNames` would miss `var` (which isn't a suitable implementation for `toString`)